### ISSUE TITLE
Not wrapping and indenting long line in function argument

### ISSUE THREAD

**Issue by cmfcmf** [prettier/prettier#12050]
**Prettier 2.5.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBnCBbOAxArlMGAS2gAoAdKAAitIEoqBeAPiuAF8AaSmgWQEMYACwB0GfgA9SANgAMszrQCM82VQBUVAA4QisACJFUMfgTilgVCUiomATgHM4MACp3TqAGYQ7GEXEUATxt7J1d3KC8fP08qLjYrEIio3wAlOE8RMFw7OwQYfyCkj280jKycvNgRWPY6BgB6KgBxVIBJfQB9AGU2gC0AUSomgFYRuko6AG5KSkhImCoAdx8Aa1QmKgFhMUkZeUVSFXkNbV0DIxMzC0TbfkdnNxLowqpgu4fw598auMVLay2ZKlDDpTLZXL5V7vGDA6JgiqQ6q1erDFrtLq9QZosZ0ECcEAQLTEaCoZCge52CBLAAK9wQZJQ-AANkt+IEyQSAEbuMCrZzdfhYAAyejgyE8LNQARAPP4fIFWnlegcyFhuBlcAwXLgABNdXrhaYHLh+E5sNFBMQoKqmbgYBB8SAhDAMMyAOpCIjwVBKsBwboM71EABu3sCyHAqE5ID00rsMBp7gc4glUplACtUBJuirmXAAIq4CDwNPM6UEpV2eORrn8HXMp1aOx6GDuoi64TIAAcChAzYg0vd7i0kebcHjIfFBIAjsX4EmiYyQPxUABaKBwPV6p15OdEPJJs2ppCS8sy6UYIhquwagmoPOF+fi0-pgkmLntztCZAAJnf7hEMyKoAMKYCeIATiMTq4NKLj1oyZ4ViAIYam0UAGrA3RgC2xIAIIYd0MCBPmZbSuw7BAA)

```sh
--parser babel # happens in Typescript as well
```

**Input:**

```jsx
someFunction(
  () => {},
  Math.max(600, (1000 * pointDistance({ x: targetTransform.e, y: targetTransform.f }, { x: transformRef.current.e, y: transformRef.current.f })) / GRID_SIZE / 55)
);

const works = Math.max(600, (1000 * pointDistance({ x: targetTransform.e, y: targetTransform.f }, { x: transformRef.current.e, y: transformRef.current.f })) / GRID_SIZE / 55)
```

**Output:**

```jsx
someFunction(() => {},
Math.max(600, (1000 * pointDistance({ x: targetTransform.e, y: targetTransform.f }, { x: transformRef.current.e, y: transformRef.current.f })) / GRID_SIZE / 55));

const works = Math.max(
  600,
  (1000 *
    pointDistance(
      { x: targetTransform.e, y: targetTransform.f },
      { x: transformRef.current.e, y: transformRef.current.f }
    )) /
    GRID_SIZE /
    55
);

```

**Expected behavior:**

The `Math.max` call in the second function argument to `someFunction` should be properly indented and wrapped similarly to how it is wrapped further down where it is assigned to the `const works` variable. It should not exceed the print width.
Note that it is wrapped correctly if I replace the first argument to `someFunction` with `1`.

// cc @tom95

**Comment by sabin-baniya** [comment]
Hey can anybody guide me on this one? I want to contribute to this one.

**Comment by alex12058** [comment]
I think this might be an issue with function argument grouping because this issue does not occur in #12426 which is an example PR that disables function argument grouping and instead breaks each argument.

You should have a look at this section of `src/language-js/print/call-arguments.js`
```js
  const shouldGroupFirst = shouldGroupFirstArg(args);
  const shouldGroupLast = shouldGroupLastArg(args, options);
  if (shouldGroupFirst || shouldGroupLast) {
    ...
  }
```

**Comment by thorn0** [comment]
fixed by #13341

**Comment by thorn0** [comment]
Duplicate of #12892
